### ISSUE TITLE
some minor quirks

### ISSUE THREAD

**Issue by megabugman** [maestrith/AHK-Studio#120]
some minor quirks you may want to fix:

1.  ctrl+q adds DC1.
in most other editors (eg. notepad++) this shortcut toggles comments. 
maybe you want to change the shortcut from ctrl+j to ctrl+q or ctrl+k?

2a. the following code opens the help file. though I'd expected to override it and show the messagebox like in other IDEs.
```
f1::
	MsgBox, hi
return
```

sometimes I get this message:
![capture](https://user-images.githubusercontent.com/17204819/28041608-64e23a6a-65ca-11e7-92cf-2914e31098cd.PNG)

2b. another issue, for some reason, selecting a command then pressing F1 does not open the contextual help file. only the help file is opened. this used to work if i remember well...

3. refresh code explorer collapses the hotkeys.
-> expected: don't collapse anything

4. closes all, open a dialog "Enter a filename for a new project"
-> expected: close all files, don't ask for a filename, rather ask where to save changes if project has not been saved yet.

**Comment by megabugman** [comment]
bored, ah in search for perfection, I decided to complete my quirky list.

5. Pressing Home executes End 
6. ctrl+left jumps the row
7. although no update available, app restarts
![bug4](https://user-images.githubusercontent.com/17204819/28225350-abe07b88-68d2-11e7-8690-4ecc083ce39c.gif)

**Comment by megabugman** [comment]
upsi, i forgot to mention that the time is not in harmony with the system settings
![time](https://user-images.githubusercontent.com/17204819/28225772-0f87a066-68d4-11e7-9cce-e9bab55ecbde.PNG)

**Comment by megabugman** [comment]
closed due to inactivity.

**Comment by maestrith** [comment]
Wow dude, just because I don't update doesn't mean you should close topics.  I am not paid to do this nor am I required to.  I will get to things when I have time.  I will NOT be responding to the "I'll just close things to get his attention" anymore.  If you want to ask about something feel free to send me a message but this is not how you will be getting my attention in future.

**Comment by megabugman** [comment]
Hey lad, by accident I clicked on one of my contributions on github and as fate willed it I landed exactly here and saw your reply. Just to put things straight, I surely do NOT strive for your attention by closing topics. I find bugs almost immediately in any software; yours is no different and a rich source in that regard. By mistake I thought you were interested learn about it in order to improve your! software. Writing bug reports is a time consuming effort, but you gave me the impression that's it's not worth worrying about, therefore I closed my reports. Besides, not everything is about money! I did not receive/expected anything for my work. Just wanted to make that clear. Good luck!
